Add "Add Asset Criticality Tag with InsightVM from Slack" (#323)
rapid7/insightconnect-workflows: workflows/Add_Asset_Criticality_Tag_in_InsightVM_from_Slack/help.md

# Description

> Set criticality tag from Slack with interactive Slack message.

Reviewer: ```suggestion
Sets criticality tag from Slack with an interactive Slack message.
```

Sample Slack Trigger Commands:

`@Rapid7 InsightConnect crit-tag-asset <hostname>`

# Key Features

* Set Asset Criticality Tag in Rapid7 InsightVM

# Requirements

* [Slack](https://insightconnect.help.rapid7.com/docs/configure-slack-for-chatops)
* [Rapid7 InsightVM](https://www.rapid7.com/products/insightvm/)

# Documentation

## Setup

Import the workflow from the Rapid7 Extension Library and proceed through the Import Workflow wizard in InsightConnect. Import plugins, create or select connections, and rename the workflow as a part of the Import Workflow wizard as necessary.

> After import, activate the workflow in order to trigger it.

Reviewer: ```suggestion
After the workflow has been import, activate the it in order to enable the ability to trigger it.
```

### Usage

*This workflow will trigger in any direct messages to your Chatbot **or** any message in a channel directed @ your Chatbot. Note the Chatbot must be in the channel in order to trigger the workflow this way.*

> To run the workflow, send a direct message to your InsightConnect Slack Chatbot or @ your Chatbot in a public channel starting with the command `crit-tag-asset`.

Reviewer: ```suggestion
To trigger the workflow, send a direct message to your InsightConnect Slack Chatbot or @ your Chatbot in a public channel starting with the command `crit-tag-asset`.
```
